youtube_dl/extractor/sonyliv.py

```diff
@@ -9,6 +9,8 @@
 from ..utils import (
     ExtractorError,
     int_or_none,
+    try_get,
+    compat_str
```

Comment:
This may work because `compat_str` is (at the moment) imported in `utils.py`, but properly it should be imported from `..compat` directly.